custom_components/smartthinq_sensors/translations/el.json

```diff
@@ -24,8 +28,7 @@
                     "use_tls_v1": "Βίαιη χρήση του πρωτοκόλλου TLSv1 (μερικές φορές απαιτείται)",
                     "exclude_dh": "Μην χρησιμοποιείτε το DH cypher (σε περίπτωση σφάλματος όπου 'το κλειδί dh είναι πολύ μικρό΄)"
                 },
```

Comment:
Could you also translate the label `use_ha_session` (present in `en.json`)

Reply:
Done.